klippy/extras/exclude_object.py

```diff
@@ -52,7 +51,11 @@ def __init__(self, config):
             desc=self.cmd_LIST_EXCLUDED_OBJECTS_help)
     def _setup_transform(self):
         if not self.next_transform:
-            logging.debug('Enabling ExcludeObject as a move transform')
+            tuning_tower = self.printer.lookup_object('tuning_tower', None)
+            if self.gcode_move.move_transform == tuning_tower:
```

Comment:
We can't peek into another object's members, so we either need to add a `get_move_transform()` method to `gcode_move`, or add the `is_testing()` method to `tuning_tower` and perform the check that way.  This would apply to the check on line 80 as well.